xdsl/interactive/app.py

```diff
@@ -216,15 +206,14 @@ def on_mount(self) -> None:
         # add titles for various widgets
         self.query_one("#input_container").border_title = "Input xDSL IR"
         self.query_one("#output_container").border_title = "Output xDSL IR"
-        self.query_one(
-            "#passes_list_view"
-        ).border_title = "Choose a pass or multiple passes to be applied."
-        self.query_one("#selected_passes").border_title = "Selected passes/query"
 
-        # initialize ListView to contain the pass options
+        # initialize Tree to contain the pass options
         for n, module_pass in ALL_PASSES:
-            self.passes_list_view.append(
-                PassListItem(Label(n), module_pass=module_pass, pass_spec=None, name=n)
+            self.passes_tree.root.add(
+                label=n,
+                data=PassListItem(
+                    Label(n), module_pass=module_pass, pass_spec=None, name=n
+                ),
```

Comment:
I'm a little confused. It seems like the tree API is different to that of the list, where Tree Data can be anything, whereas the List requires a ListItem. Now we're duplicating the label, which seems unnecessary. Can you please exctract the "data" part of the PassListItem, separate from the label, into a NamedTuple, and pass that to the tree node instead?